src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,46 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents an addition user command.
+ * Inherits from command class.
+ */
+public class AddCommand extends Command {
+    /**
+     * Constructs AddCommand object.
+     *
+     * @param cmd task command to be added.
+     */
+    public AddCommand(String cmd) {
+        super(cmd);
+    }
+
+    /**
+     * Executes command.
+     * Adds task command into TaskList.
+     *
+     * @param tasks list of tasks within chat bot.
+     * @param ui user interface of chat bot.
+     * @param storage file directory manager.
+     * @throws DukeException If unable to add task.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        ui.showAdded(tasks.add(line), tasks.getSize());
+        super.execute(tasks, ui, storage);
+    }
+
+    /**
+     * Returns if command exits program.
+     *
+     * @return if command exists program.
```

Comment:
`@return false` would be more apt and indicative of the method

Reply:
Agreed, it would be clearer for readers.